sdk/src/snapshot.rs

```diff
@@ -0,0 +1,160 @@
+use std::{fmt, str::FromStr};
+
+use serde::{Deserialize, Serialize};
+
+use crate::error::IggyError;
+
+/// Enum representing the different types of system snapshots that can be taken.
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
+pub enum SystemSnapshotType {
```

Comment:
How Iggy server logs can be collected using those snapshot types? I think server logs are a key asset in this functionality.

Reply:
Yes, that's correct. Currently, we can only collect a limited amount of data using the CLI commands that are already available in the iggy-server image. We've agreed with @hubcio to start with a minimal implementation that can be extended in the future. The next step will be to add the journalctl utility to the image and expand the SystemSnapshotType so that it can directly collect logs from the iggy-server

Reply:
I initially thought you meant downloading all logs, including system logs. After discussing with @hubcio, I realized we only need the server logs from local_data. I'll add this command in the current MR

Reply:
added the new command: server_logs